### DISCUSSION TITLE
能开设一个QQ或者Telegram群组吗

### DISCUSSION THREAD

**Discussion by zerosnowe** [ywmoyue/biliuwp-lite#732] [General]
有时候使用IM即时通讯软件来讨论会更方便

**Comment by ywmoyue** [comment]
目前没有多的时间管理一个群，以后有空可能会搞

**Reply by zerosnowe** [reply]
可不可以让用户自发组建群聊，这样就不用花时间组建👏

**Reply by ProJend** [reply]
> 可不可以让用户自发组建群聊，这样就不用花时间组建👏

自发就乱套了，能信任的用户只有owner本人

**Reply by Yumeyo23** [reply]
> > 可不可以让用户自发组建群聊，这样就不用花时间组建👏
> 
> 自发就乱套了，能信任的用户只有owner本人

不认为，可能他的意思就类似大学的班级群一样，老师和辅导员只是管理员，而群主可能是班上的同学，
当然确实最好还是owner当群主，其实作者担心没时间管理，我觉得大可以把管理的任务交给我们这些本项目支持者，我们也许不一定能在程序代码上帮大忙，但是收集问题，管理群成员发言违规，这些最基本的，我觉得我们是可以做到的，至少我是每天都在水QQ我是很乐意做这个事儿的哈哈哈哈

**Reply by ProJend** [reply]
> > > 可不可以让用户自发组建群聊，这样就不用花时间组建👏
> 
> > 
> 
> > 自发就乱套了，能信任的用户只有owner本人
> 
> 
> 
> 不认为，可能他的意思就类似大学的班级群一样，老师和辅导员只是管理员，而群主可能是班上的同学，
> 
> 当然确实最好还是owner当群主，其实作者担心没时间管理，我觉得大可以把管理的任务交给我们这些本项目支持者，我们也许不一定能在程序代码上帮大忙，但是收集问题，管理群成员发言违规，这些最基本的，我觉得我们是可以做到的，至少我是每天都在水QQ我是很乐意做这个事儿的哈哈哈哈

你已经提到让作者独立完成了，这个和用户自发创建群聊无关

**Comment by zerosnowe** [comment]
感谢答复，不过可以让一些用户自发管理群聊，能缩短管理的精力
________________________________
发件人: ywmoyue ***@***.***>
发送时间: Tuesday, July 30, 2024 11:33:35 AM
收件人: ywmoyue/biliuwp-lite ***@***.***>
抄送: zhatuganjun ***@***.***>; Author ***@***.***>
主题: Re: [ywmoyue/biliuwp-lite] 能开设一个QQ或者Telegram群组吗 (Discussion #732)


目前没有多的时间管理一个群，以后有空可能会搞

―

**Comment by Yumeyo23** [comment]
插眼，支持，这个项目真心很好，去年就在反馈BUG作者大大也一直在跟进，虽然我不懂编程，但我也很想帮上能帮的忙

**Comment by zerosnowe** [comment]
确实非常好，一直在修复重构橙子大佬留下的bug，用心

________________________________
发件人: youhunluanma ***@***.***>
发送时间: Thursday, August 1, 2024 5:20:49 AM
收件人: ywmoyue/biliuwp-lite ***@***.***>
抄送: zhatuganjun ***@***.***>; Author ***@***.***>
主题: Re: [ywmoyue/biliuwp-lite] 能开设一个QQ或者Telegram群组吗 (Discussion #732)


插眼，支持，这个项目真心很好，去年就在反馈BUG作者大大也一直在跟进，虽然我不懂编程，但我也很想帮上能帮的忙

―
